integration/db/src/main/java/com/hbc/pms/integration/db/entities/SensorConfigurationFigure.java

```diff
@@ -0,0 +1,29 @@
+package com.hbc.pms.integration.db.entities;
+
+import jakarta.persistence.*;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Table(name = "sensor_configuration_figure")
+@Builder
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class SensorConfigurationFigure extends BaseEntity {
+
+  @Column(name = "name")
+  private String name;
+
+  @Column(name = "data_type")

```

Comment:
No need to specific the name on every column, use the naming strategy  CamelCaseToUnderscoresNamingStrategy